base/errorshow.jl

```diff
@@ -272,8 +272,7 @@ function showerror(io::IO, ex::MethodError)
         if !isempty(kwargs)
             print(io, "; ")
             for (i, (k, v)) in enumerate(kwargs)
-                print(io, k, "=")
-                show(IOContext(io, :limit => true), v)
+                print(io, k, "::", typeof(v))
```

Comment:
Should this be 'showarg'?

Reply:
So `f(::Float64; v=view(::Matrix{Float64}, :, :) with eltype Float64)`. That doesn't look right?